web_external/js/views/body/AnalysisPanel.js

```diff
@@ -10,7 +10,7 @@
         return name  + 'Widget';
     };
 
-    minerva.views.AnalysisPanel = minerva.View.extend({
+    minerva.views.AnalysisPanel = minerva.views.Panel.extend({
```

Comment:
Since we are asking panels to extend 'minerva.views.Panel' instead of 'minerva.View' now, can you make the below happen automatically in Panel.js, and then specific panel extensions wouldn't need to do it?

    _.extend(this.events, minerva.views.Panel.prototype.events);